### PR TITLE
add a `revnumber` doc attribute when publishing docs

### PR DESCRIPTION
**Pull request by Ladicek**
This approach is a bit dirty and I'm open to better solutions.

### CODE REVIEW

File: ci/Jenkinsfile [hunk 1]
```diff
@@ -2,6 +2,7 @@ node("docsbuilder-maven") {
   checkout scm
   stage("Build Documentation") {
     sh "pwd && ls -l"
+    sh "echo :revnumber: $(git rev-parse --short=10 HEAD) >> ./docs/topics/templates/document-attributes.adoc"
```
Comment (Ladicek) [inline ci/Jenkinsfile]: Actually I think there's an environment variable that Jenkins exposes that contains the revision. That's better than `git rev-parse`, even though it probably isn't nicely shortened. Please hold this PR, I'll amend on Monday.

Comment (rhoads-zach) [inline ci/Jenkinsfile]: No problem, thanks @Ladicek

**Comment by Ladicek** [comment]
Actually it turns out that current Git revision number isn't available in a Jenkins pipeline (see e.g. https://issues.jenkins-ci.org/browse/JENKINS-34455 and https://github.com/jenkinsci/pipeline-examples/tree/master/pipeline-examples/gitcommit). I had to make a small fix, tested it and this is good to go.

**Comment by rhoads-zach** [comment]
Thanks @Ladicek 

LGTM, merging.

**Comment by rhoads-zach** [comment]
merged: https://github.com/openshiftio/appdev-documentation/commit/76210191cb138654b13589f4b8c0fca22d487e9e

**Pull request closed without merging**